lib/platform/github/index.ts

```diff
@@ -1202,12 +1202,16 @@ export async function ensureIssue({
       }
       if (shouldReOpen) {
         logger.debug('Patching issue');
+        const data: Record<string, unknown> = { body, state: 'open', title };
+        if (labels) {
```

Comment:
@viceice Should be `if (labels?.length) {` ?

Reply:
not sure, is `labels`  always defined? i thought it's undefined by default 🤔

Reply:
Something's not working. Can we go with a more conservative approach?

Reply:
IÄll try to run some debug sessions, to see why it's again removing the labels when it shouldn't. Maybe we've a bug somewhere.

Reply:
My guess is that the platform layer is receiving an empty array